eclipse-collections-api/src/main/java/org/eclipse/collections/api/set/sorted/MutableSortedSet.java

```diff
@@ -166,6 +167,9 @@
     <S> MutableList<Pair<T, S>> zip(Iterable<S> that);
 
     @Override
+    <S, Z> OrderedIterable<Z> zipWith(Iterable<S> that, Function2<T, S, Z> function);
```

Comment:
Did you intend to change the return type here to MutableSortedSet? As it stands, this override is redundant.